docs/team/weijie96.md

```diff
@@ -12,26 +12,45 @@ It is written in Java, and has about 10 kLoC.
 Given below are my contributions to the project.
 
 * **New Feature**: Added the ability to archive clients.
-  * What it does: **Coming Soon**
-  * Justification: **Coming Soon**
-  * Highlights: **Coming Soon**
+  * What it does: It allows the user to add active clients into the archive. The opposite can be done as well; archived clients can be made active again.
+  To support the archiving feature, users can also view the archive, as well as to switch back to view the active clients.
+  * Justification: This feature improves the product significantly because a user can add clients which are not currently relevant to the user into the archive.
```

Comment:
For this, I think it may be better if you differentiate between insurance agent and client. You used user a few times, it might be confusing, besides linking it back to the target audience would help to solidify this case.

Reply:
I would make the change, but actually our DG and UG references "user" most of the time; but for PPP I'll update